tasks/static-files.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Set variables
+  set_fact:
+    "{{ inner_item.key }}": "{{ inner_item.value }}"
+  with_dict: "{{ item }}"
```

Comment:
This will use non-deterministic ordering.  Does this affect the deploy status?  i.e. if I run this twice, I should always get a fully green run the second time.

Reply:
set_fact is always ok/green

Reply:
Ok, sounds good